C7GameData/Tile.cs

```diff
@@ -50,6 +50,22 @@ public MapUnit findTopDefender()
         
         public static Tile NONE = new Tile();
 
+        public bool neighborsCoast() {
```

Comment:
Minor nitpick, but the indentation here is a bit wonky. At some point, someone will probably have to bite the bullet an go through and standardize all the many different styles used throughout the different files, but that's less important right now than actually writing code.

Reply:
Gee, that is wonky.  A mix of spaces and tabs in the same line even!  Fired it up in Notepad++ so I could see them differently, and converted the whole file to tabs.

I think part of the reason we have this problem is that VSCode has a setting "Editor: Detect Indentation" that is enabled by default.  Puppeteer and I had agreed to set tabs, and figured out how to do it, we thought, but I at least didn't know that I was only setting it for one file, and it was auto-detecting based on other files.

I've un-checked that setting now, so going forward my code should be tab-indented.